### DISCUSSION TITLE
Slingbox PRO (SB200-100) recovery status

### DISCUSSION THREAD

**Discussion by flipper2222222** [GerryDazoo/Slinger#592] [General]
hi all,

can anyone enlighten me if the
Slingbox PRO (SB200-100)
can be brought online and what is the process.

I bought it from eBay, don't know username and password, don't have any other details except MAC address that I have blocked from accessing the internet

it has not had any network access since before 9 November 2022

thanks in advance.

**Comment by lead2crm** [comment]
1. Get your password by following these instructions: https://github.com/GerryDazoo/Slinger/blob/main/sb-pwd.md
2. Edit the config.ini file with your password and select what type of slingbox you have, these are int he first few lines.
3. If you have a Windows PC, just run the slingbox_server.exe --much easier.
4. Download VLC Player and connect to stream
5. the stream address is your home ROUTER IP address like: http://192.168.x.x:8080/slingbox ---change out to your IP address.

**Reply by lead2crm** [reply]
This will get your streaming running locally. I still havent figured out how to get this runnign outside of my local network (I need help with firewall and port forwarding)

**Reply by flipper2222222** [reply]
thanks a lot  , it was step one I was missing, I have 2 slingboxes working, but just got a new box with unknown credentials, , I will try Gerry's postman method

**Reply by flipper2222222** [reply]
I am confused on step 1 the postman method. 
is this method usable for a slingbox that was bought of eBay and only became my property after 9 Nov 2022?
thanks

**Reply by GerryDazoo** [reply]
No it's not. This method recovers the password(s) for Slingboxes that had been registered to your account pre N9.

**Reply by flipper2222222** [reply]
What’s possible with this sling box that I have from eBay? 
anything now? 
If not anything likely in the future?

**Reply by GerryDazoo** [reply]
You can do a factory reset to set the default password back to "admin". I don't have a 200 to test with but I would expect the instructions here to revive a bricked Pro-HD to be applicable.

**Comment by skijohn877** [comment]
You need to add port forwarding on your router, source port would be 8080, target ip would be your internal ip (192.168.x.x) were you have slingbox_server.exe running with target port being 8080. Not sure about windows firewall, but you can always disable it.

**Comment by lead2crm** [comment]
Doesn't using port 8080 leave this more vulnerable to hacking attempts?

**Comment by skijohn877** [comment]
Yes, what I do is make the external port something very high like 48080, most port scanners will not scan for ports that high.

**Reply by lead2crm** [reply]
@skijohn877 The picklist for "Device" has Slingbox500 pre-configured from when Slingbox worked from the service prior to bricking these. Can I use that as selected or do I need to click the "IP Address" radio button and put iin the IP Address directly to the Slingbox?  

I have no idea how to configure the rest of this screenshot below... i.e. "Port Start and Port End ...why isn't is just one static port?

![image](https://user-images.githubusercontent.com/20845641/236701025-cd0d973c-075e-43c5-95a1-59d879694fb6.png)

**Comment by skijohn877** [comment]
You would use the ip address radio button under the Local Port and IP, then enter the ip address were the sling_server.exe is running (not the address of the slingbox), port start and end would be 8080, WAN Ports would be 8080 or like I said earlier make them something very high.

**Reply by lead2crm** [reply]
This worked and I selected a port number of my choosing. The only other addition for those reading this thread is you need to update the port number in the config.ini file (it's toward the bottom).

I was able to keep Windows Firewall Defender intact, just gave extra permission to sling_server.exe to be "Public", as "Private" was already checked.

Is there anything else I need to do to help tighten up the security on this? Is there much to even be worried about? The PC that is running this is literally useless beyond this feature and has no cool information or passwords or Networking Connections--not even a printer.

**Comment by flipper2222222** [comment]
thanks to all others that contributed to this discussion.
